api/v1alpha1/gateway_types.go

```diff
@@ -167,7 +223,7 @@ type ListenerTLS struct {
 	// values.
 	//
 	// +optional
-	MinimumVersion *string `json:"minimumVersion"`
+	MinimumVersion *string `json:"minimumVersion,omitempty"`
```

Comment:
Hmm, are there any requirements on implementations if the minimum version is omitted? Maybe track a separate issue to discuss?